assemblyline/odm/models/service_delta.py

```diff
@@ -16,6 +16,7 @@ class DockerConfigDelta(odm.Model):
     image = odm.Optional(odm.Keyword())  # The docker image and tag, optionally including registry in the normal way
     registry_username = odm.Optional(odm.Keyword())  # The username to use when pulling the image
     registry_password = odm.Optional(odm.Keyword())  # The password or token to use when pulling the image
+    registry_type = odm.Enum(values=["docker", "harbor"], default='docker')  # The type of registry (Docker, Harbor)
```

Comment:
All service_delta fields MUST be optional with no default value set

Reply:
fixed